src/styles/main.scss

```diff
@@ -1 +1,6 @@
-// put code here
+@import "../utils/variables";
+@import "../utils/reset";
```

Comment:
the reset import should be used in 1st line because rest of files can depend on that, so please move this declaration up